.github/workflows/ci.yml

```diff
@@ -18,14 +18,13 @@ env:
   # An envar that signals to tests we are executing in the CI environment
   CONTINUOUS_INTEGRATION: true
   # allow overriding Maven command
-  MAVEN: ./mvnw --offline
-  # maven.wagon.rto is in millis, defaults to 30m
-  MAVEN_OPTS: "-Xmx512M -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
```

Comment:
Why `-Xmx512M -XX:+ExitOnOutOfMemoryError` was removed?
is this related to removal of the go-offline?

Reply:
https://github.com/trinodb/trino/pull/21256